lib/foreman_tasks/dynflow/configuration.rb

```diff
@@ -1,4 +1,4 @@
-require File.expand_path('lib/foreman/dynflow/configuration', Rails.root)
+require File.expand_path('lib/foreman/dynflow/configuration', ::Rails.root)
```

Comment:
FYI This actually should be `Rails.root.join('lib', 'foreman' ...)`. But it's unrelated to this PR :)